champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/gladiator/sword/BlockToss.java

```diff
@@ -0,0 +1,379 @@
+    }
+    }
+
+    private void showCharge(Player player, BlockToss.BlockData charge) {
+        // Sound
+        if (!UtilTime.elapsed(charge.getLastSound(), 150)) {
+            return;
+        }
+        player.playSound(player.getEyeLocation(), Sound.BLOCK_NOTE_BLOCK_PLING, 0.5f, 0.5f + (0.5f * (float) charge.getCharge()));
+        charge.setLastSound(System.currentTimeMillis());
+    }
+
+    @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+    public void cancelCooldown(PlayerUseSkillEvent event) {
+        if (event.getSkill() == this && charging.containsKey(event.getPlayer() )) {
+            event.setCancelled(true); // Cancel cooldown or ability use if they're charging to allow them to release
+        }
+    }
+
+
+    @UpdateEvent
+    public void updateCharge() {
+        // Charge check
+        Iterator<Player> iterator = charging.keySet().iterator();
+        while (iterator.hasNext()) {
+            Player player = iterator.next();
+                BlockToss.BlockData charge = charging.get(player);
+                if (player != null) {
+                    int level = getLevel(player);
+                    // Remove if they no longer have the skill
+                    if (level <= 0) {
+                        iterator.remove();
+                        continue;
+                    }
+                    //Remove if they don't have a block
+                    if (findActiveBlock(player)==null) {
+                        iterator.remove();
+                        continue;
+                    }
+                    // Check if they still are blocking and charge
+                    if (player.isHandRaised()) {
+                        championsManager.getCooldowns().removeCooldown(player, getName(), true);
+                        // Check for sword hold status
+                        if (!UtilPlayer.isHoldingItem(player, SkillWeapons.SWORDS)) {
+                            findActiveBlock(player).getVehicle().remove();
+                            findActiveBlock(player).remove();
+                            iterator.remove(); // Otherwise, remove
+                        }
+                        charge.tick();
+                        // Cues
+                        showCharge(player, charge);
+                        continue;
+                    }
+                    if (UtilPlayer.isHoldingItem(player, SkillWeapons.SWORDS)) {
+                        setCharge(charge.getCharge());
+                        findActiveBlock(player).getVehicle().remove();
+                        thrownBlock(player, (float) charge.getCharge(), findActiveBlock(player));
+                        activeBlocks.put(findActiveBlock(player), false);
+                        charging.remove(player);
+
+                    }
+            }
+        }
+    }
+
+    @Override
+    public void loadSkillConfig(){
+        baseCharge = getConfig("baseCharge", 60.0, Double.class);
```

Comment:
Base charge should be 0.60 and calculated accordingly to fit with other percentage config values.